entries/c/combell.com.json

```
@@ -0,0 +1,14 @@
+{
+  "Combell": {
+    "domain": "combell.com",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://www.combell.com/en/help/kb/how-do-i-enable-two-factor-authentication/",
+    "categories": [
+      "domains",
+      "hosting",
+      "cloud"
```

I don't think "serverless" cloud computing is something Combell provides.
```suggestion
      "domains",
      "hosting"
```